src/league.js

```diff
@@ -647,12 +643,13 @@ function League(options) {
 }
 
 function getAllLeagues(config) {
-    var leagues = [];
     var all_league_configs = config['leagues'] || {};
-    _.each(_.keys(all_league_configs), function(key) {
-        leagues.push(getLeague(key, config));
-    });
-    return leagues;
+    return _(all_league_configs)
```

Comment:
I suppose I should get used to this syntax. I would not have written it this way. I tend to code in a more procedural style - all that C++98...